RELEASE.md

```diff
@@ -8,10 +8,14 @@ Please follow the established format:
 
 # Release 6.6.0
 
+## Major features and improvements
+
+- Make Kedro-Viz shareable via a hosted URL. (#1487)
+
 ## Bug fixes and other changes
 
-- Fix for Kedro Viz Connection Error (#1507)
-- Fix display of modular pipeline nodes that are associated with tags (#1542)
+- Fix for Kedro-Viz Connection Error. (#1507)
+- Fix display of modular pipeline nodes that are associated with tags. (#1542)
 
 # Release 6.5.1
```

Comment:
Not sure if we want to remove this (6.5.1) release note here but commenting here before we merge it to main